Issue compiling with tsc

**Describe the bug**
A clear and concise description of what the bug is.

**Installed libraries and their versions**

* adminjs@5.5.1
* @adminjs/express@4.0.1
* @adminjs/prisma@1.0.2

**To Reproduce**
Run `tsc`

**Expected behavior**
The project to compile

**Screenshots**
```
yarn run v1.22.5
$ tsc src/api/index.ts --outDir dist --module commonjs --esModuleInterop
node_modules/@types/react/index.d.ts(3092,19): error TS2320: Interface 'ElementClass' cannot simultaneously extend types 'Component<any, {}, any>' and 'Component<any, {}, any>'.
  Named property 'props' of types 'Component<any, {}, any>' and 'Component<any, {}, any>' are not identical.
node_modules/@types/react/index.d.ts(3092,19): error TS2320: Interface 'ElementClass' cannot simultaneously extend types 'Component<any, {}, any>' and 'Component<any, {}, any>'.
  Named property 'refs' of types 'Component<any, {}, any>' and 'Component<any, {}, any>' are not identical.
node_modules/@types/react/index.d.ts(3100,14): error TS2300: Duplicate identifier 'LibraryManagedAttributes'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3092,14): error TS2300: Duplicate identifier 'LibraryManagedAttributes'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3105,13): error TS2717: Subsequent property declarations must have the same type.  Property 'a' must be of type 'DetailedHTMLProps<AnchorHTMLAttributes<HTMLAnchorElement>, HTMLAnchorElement>', but here has type 'DetailedHTMLProps<AnchorHTMLAttributes<HTMLAnchorElement>, HTMLAnchorElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3106,13): error TS2717: Subsequent property declarations must have the same type.  Property 'abbr' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3107,13): error TS2717: Subsequent property declarations must have the same type.  Property 'address' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3108,13): error TS2717: Subsequent property declarations must have the same type.  Property 'area' must be of type 'DetailedHTMLProps<AreaHTMLAttributes<HTMLAreaElement>, HTMLAreaElement>', but here has type 'DetailedHTMLProps<AreaHTMLAttributes<HTMLAreaElement>, HTMLAreaElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3109,13): error TS2717: Subsequent property declarations must have the same type.  Property 'article' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3110,13): error TS2717: Subsequent property declarations must have the same type.  Property 'aside' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3111,13): error TS2717: Subsequent property declarations must have the same type.  Property 'audio' must be of type 'DetailedHTMLProps<AudioHTMLAttributes<HTMLAudioElement>, HTMLAudioElement>', but here has type 'DetailedHTMLProps<AudioHTMLAttributes<HTMLAudioElement>, HTMLAudioElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3112,13): error TS2717: Subsequent property declarations must have the same type.  Property 'b' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3113,13): error TS2717: Subsequent property declarations must have the same type.  Property 'base' must be of type 'DetailedHTMLProps<BaseHTMLAttributes<HTMLBaseElement>, HTMLBaseElement>', but here has type 'DetailedHTMLProps<BaseHTMLAttributes<HTMLBaseElement>, HTMLBaseElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3114,13): error TS2717: Subsequent property declarations must have the same type.  Property 'bdi' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3115,13): error TS2717: Subsequent property declarations must have the same type.  Property 'bdo' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3116,13): error TS2717: Subsequent property declarations must have the same type.  Property 'big' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3117,13): error TS2717: Subsequent property declarations must have the same type.  Property 'blockquote' must be of type 'DetailedHTMLProps<BlockquoteHTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<BlockquoteHTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3118,13): error TS2717: Subsequent property declarations must have the same type.  Property 'body' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLBodyElement>, HTMLBodyElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLBodyElement>, HTMLBodyElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3119,13): error TS2717: Subsequent property declarations must have the same type.  Property 'br' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLBRElement>, HTMLBRElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLBRElement>, HTMLBRElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3120,13): error TS2717: Subsequent property declarations must have the same type.  Property 'button' must be of type 'DetailedHTMLProps<ButtonHTMLAttributes<HTMLButtonElement>, HTMLButtonElement>', but here has type 'DetailedHTMLProps<ButtonHTMLAttributes<HTMLButtonElement>, HTMLButtonElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3121,13): error TS2717: Subsequent property declarations must have the same type.  Property 'canvas' must be of type 'DetailedHTMLProps<CanvasHTMLAttributes<HTMLCanvasElement>, HTMLCanvasElement>', but here has type 'DetailedHTMLProps<CanvasHTMLAttributes<HTMLCanvasElement>, HTMLCanvasElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3122,13): error TS2717: Subsequent property declarations must have the same type.  Property 'caption' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3123,13): error TS2717: Subsequent property declarations must have the same type.  Property 'cite' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3124,13): error TS2717: Subsequent property declarations must have the same type.  Property 'code' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3125,13): error TS2717: Subsequent property declarations must have the same type.  Property 'col' must be of type 'DetailedHTMLProps<ColHTMLAttributes<HTMLTableColElement>, HTMLTableColElement>', but here has type 'DetailedHTMLProps<ColHTMLAttributes<HTMLTableColElement>, HTMLTableColElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3126,13): error TS2717: Subsequent property declarations must have the same type.  Property 'colgroup' must be of type 'DetailedHTMLProps<ColgroupHTMLAttributes<HTMLTableColElement>, HTMLTableColElement>', but here has type 'DetailedHTMLProps<ColgroupHTMLAttributes<HTMLTableColElement>, HTMLTableColElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3127,13): error TS2717: Subsequent property declarations must have the same type.  Property 'data' must be of type 'DetailedHTMLProps<DataHTMLAttributes<HTMLDataElement>, HTMLDataElement>', but here has type 'DetailedHTMLProps<DataHTMLAttributes<HTMLDataElement>, HTMLDataElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3128,13): error TS2717: Subsequent property declarations must have the same type.  Property 'datalist' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLDataListElement>, HTMLDataListElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLDataListElement>, HTMLDataListElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3129,13): error TS2717: Subsequent property declarations must have the same type.  Property 'dd' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3130,13): error TS2717: Subsequent property declarations must have the same type.  Property 'del' must be of type 'DetailedHTMLProps<DelHTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<DelHTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3131,13): error TS2717: Subsequent property declarations must have the same type.  Property 'details' must be of type 'DetailedHTMLProps<DetailsHTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<DetailsHTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3132,13): error TS2717: Subsequent property declarations must have the same type.  Property 'dfn' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3133,13): error TS2717: Subsequent property declarations must have the same type.  Property 'dialog' must be of type 'DetailedHTMLProps<DialogHTMLAttributes<HTMLDialogElement>, HTMLDialogElement>', but here has type 'DetailedHTMLProps<DialogHTMLAttributes<HTMLDialogElement>, HTMLDialogElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3134,13): error TS2717: Subsequent property declarations must have the same type.  Property 'div' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3135,13): error TS2717: Subsequent property declarations must have the same type.  Property 'dl' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLDListElement>, HTMLDListElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLDListElement>, HTMLDListElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3136,13): error TS2717: Subsequent property declarations must have the same type.  Property 'dt' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3137,13): error TS2717: Subsequent property declarations must have the same type.  Property 'em' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3138,13): error TS2717: Subsequent property declarations must have the same type.  Property 'embed' must be of type 'DetailedHTMLProps<EmbedHTMLAttributes<HTMLEmbedElement>, HTMLEmbedElement>', but here has type 'DetailedHTMLProps<EmbedHTMLAttributes<HTMLEmbedElement>, HTMLEmbedElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3139,13): error TS2717: Subsequent property declarations must have the same type.  Property 'fieldset' must be of type 'DetailedHTMLProps<FieldsetHTMLAttributes<HTMLFieldSetElement>, HTMLFieldSetElement>', but here has type 'DetailedHTMLProps<FieldsetHTMLAttributes<HTMLFieldSetElement>, HTMLFieldSetElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3140,13): error TS2717: Subsequent property declarations must have the same type.  Property 'figcaption' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3141,13): error TS2717: Subsequent property declarations must have the same type.  Property 'figure' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3142,13): error TS2717: Subsequent property declarations must have the same type.  Property 'footer' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3143,13): error TS2717: Subsequent property declarations must have the same type.  Property 'form' must be of type 'DetailedHTMLProps<FormHTMLAttributes<HTMLFormElement>, HTMLFormElement>', but here has type 'DetailedHTMLProps<FormHTMLAttributes<HTMLFormElement>, HTMLFormElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3144,13): error TS2717: Subsequent property declarations must have the same type.  Property 'h1' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLHeadingElement>, HTMLHeadingElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLHeadingElement>, HTMLHeadingElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3145,13): error TS2717: Subsequent property declarations must have the same type.  Property 'h2' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLHeadingElement>, HTMLHeadingElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLHeadingElement>, HTMLHeadingElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3146,13): error TS2717: Subsequent property declarations must have the same type.  Property 'h3' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLHeadingElement>, HTMLHeadingElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLHeadingElement>, HTMLHeadingElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3147,13): error TS2717: Subsequent property declarations must have the same type.  Property 'h4' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLHeadingElement>, HTMLHeadingElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLHeadingElement>, HTMLHeadingElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3148,13): error TS2717: Subsequent property declarations must have the same type.  Property 'h5' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLHeadingElement>, HTMLHeadingElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLHeadingElement>, HTMLHeadingElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3149,13): error TS2717: Subsequent property declarations must have the same type.  Property 'h6' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLHeadingElement>, HTMLHeadingElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLHeadingElement>, HTMLHeadingElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3150,13): error TS2717: Subsequent property declarations must have the same type.  Property 'head' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLHeadElement>, HTMLHeadElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLHeadElement>, HTMLHeadElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3151,13): error TS2717: Subsequent property declarations must have the same type.  Property 'header' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3152,13): error TS2717: Subsequent property declarations must have the same type.  Property 'hgroup' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3153,13): error TS2717: Subsequent property declarations must have the same type.  Property 'hr' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLHRElement>, HTMLHRElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLHRElement>, HTMLHRElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3154,13): error TS2717: Subsequent property declarations must have the same type.  Property 'html' must be of type 'DetailedHTMLProps<HtmlHTMLAttributes<HTMLHtmlElement>, HTMLHtmlElement>', but here has type 'DetailedHTMLProps<HtmlHTMLAttributes<HTMLHtmlElement>, HTMLHtmlElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3155,13): error TS2717: Subsequent property declarations must have the same type.  Property 'i' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3156,13): error TS2717: Subsequent property declarations must have the same type.  Property 'iframe' must be of type 'DetailedHTMLProps<IframeHTMLAttributes<HTMLIFrameElement>, HTMLIFrameElement>', but here has type 'DetailedHTMLProps<IframeHTMLAttributes<HTMLIFrameElement>, HTMLIFrameElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3157,13): error TS2717: Subsequent property declarations must have the same type.  Property 'img' must be of type 'DetailedHTMLProps<ImgHTMLAttributes<HTMLImageElement>, HTMLImageElement>', but here has type 'DetailedHTMLProps<ImgHTMLAttributes<HTMLImageElement>, HTMLImageElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3158,13): error TS2717: Subsequent property declarations must have the same type.  Property 'input' must be of type 'DetailedHTMLProps<InputHTMLAttributes<HTMLInputElement>, HTMLInputElement>', but here has type 'DetailedHTMLProps<InputHTMLAttributes<HTMLInputElement>, HTMLInputElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3159,13): error TS2717: Subsequent property declarations must have the same type.  Property 'ins' must be of type 'DetailedHTMLProps<InsHTMLAttributes<HTMLModElement>, HTMLModElement>', but here has type 'DetailedHTMLProps<InsHTMLAttributes<HTMLModElement>, HTMLModElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3160,13): error TS2717: Subsequent property declarations must have the same type.  Property 'kbd' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3161,13): error TS2717: Subsequent property declarations must have the same type.  Property 'keygen' must be of type 'DetailedHTMLProps<KeygenHTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<KeygenHTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3162,13): error TS2717: Subsequent property declarations must have the same type.  Property 'label' must be of type 'DetailedHTMLProps<LabelHTMLAttributes<HTMLLabelElement>, HTMLLabelElement>', but here has type 'DetailedHTMLProps<LabelHTMLAttributes<HTMLLabelElement>, HTMLLabelElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3163,13): error TS2717: Subsequent property declarations must have the same type.  Property 'legend' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLLegendElement>, HTMLLegendElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLLegendElement>, HTMLLegendElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3164,13): error TS2717: Subsequent property declarations must have the same type.  Property 'li' must be of type 'DetailedHTMLProps<LiHTMLAttributes<HTMLLIElement>, HTMLLIElement>', but here has type 'DetailedHTMLProps<LiHTMLAttributes<HTMLLIElement>, HTMLLIElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3165,13): error TS2717: Subsequent property declarations must have the same type.  Property 'link' must be of type 'DetailedHTMLProps<LinkHTMLAttributes<HTMLLinkElement>, HTMLLinkElement>', but here has type 'DetailedHTMLProps<LinkHTMLAttributes<HTMLLinkElement>, HTMLLinkElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3166,13): error TS2717: Subsequent property declarations must have the same type.  Property 'main' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3167,13): error TS2717: Subsequent property declarations must have the same type.  Property 'map' must be of type 'DetailedHTMLProps<MapHTMLAttributes<HTMLMapElement>, HTMLMapElement>', but here has type 'DetailedHTMLProps<MapHTMLAttributes<HTMLMapElement>, HTMLMapElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3168,13): error TS2717: Subsequent property declarations must have the same type.  Property 'mark' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3169,13): error TS2717: Subsequent property declarations must have the same type.  Property 'menu' must be of type 'DetailedHTMLProps<MenuHTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<MenuHTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3170,13): error TS2717: Subsequent property declarations must have the same type.  Property 'menuitem' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3171,13): error TS2717: Subsequent property declarations must have the same type.  Property 'meta' must be of type 'DetailedHTMLProps<MetaHTMLAttributes<HTMLMetaElement>, HTMLMetaElement>', but here has type 'DetailedHTMLProps<MetaHTMLAttributes<HTMLMetaElement>, HTMLMetaElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3172,13): error TS2717: Subsequent property declarations must have the same type.  Property 'meter' must be of type 'DetailedHTMLProps<MeterHTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<MeterHTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3173,13): error TS2717: Subsequent property declarations must have the same type.  Property 'nav' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3174,13): error TS2717: Subsequent property declarations must have the same type.  Property 'noindex' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3175,13): error TS2717: Subsequent property declarations must have the same type.  Property 'noscript' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3176,13): error TS2717: Subsequent property declarations must have the same type.  Property 'object' must be of type 'DetailedHTMLProps<ObjectHTMLAttributes<HTMLObjectElement>, HTMLObjectElement>', but here has type 'DetailedHTMLProps<ObjectHTMLAttributes<HTMLObjectElement>, HTMLObjectElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3177,13): error TS2717: Subsequent property declarations must have the same type.  Property 'ol' must be of type 'DetailedHTMLProps<OlHTMLAttributes<HTMLOListElement>, HTMLOListElement>', but here has type 'DetailedHTMLProps<OlHTMLAttributes<HTMLOListElement>, HTMLOListElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3178,13): error TS2717: Subsequent property declarations must have the same type.  Property 'optgroup' must be of type 'DetailedHTMLProps<OptgroupHTMLAttributes<HTMLOptGroupElement>, HTMLOptGroupElement>', but here has type 'DetailedHTMLProps<OptgroupHTMLAttributes<HTMLOptGroupElement>, HTMLOptGroupElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3179,13): error TS2717: Subsequent property declarations must have the same type.  Property 'option' must be of type 'DetailedHTMLProps<OptionHTMLAttributes<HTMLOptionElement>, HTMLOptionElement>', but here has type 'DetailedHTMLProps<OptionHTMLAttributes<HTMLOptionElement>, HTMLOptionElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3180,13): error TS2717: Subsequent property declarations must have the same type.  Property 'output' must be of type 'DetailedHTMLProps<OutputHTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<OutputHTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3181,13): error TS2717: Subsequent property declarations must have the same type.  Property 'p' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLParagraphElement>, HTMLParagraphElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLParagraphElement>, HTMLParagraphElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3182,13): error TS2717: Subsequent property declarations must have the same type.  Property 'param' must be of type 'DetailedHTMLProps<ParamHTMLAttributes<HTMLParamElement>, HTMLParamElement>', but here has type 'DetailedHTMLProps<ParamHTMLAttributes<HTMLParamElement>, HTMLParamElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3183,13): error TS2717: Subsequent property declarations must have the same type.  Property 'picture' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3184,13): error TS2717: Subsequent property declarations must have the same type.  Property 'pre' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLPreElement>, HTMLPreElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLPreElement>, HTMLPreElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3185,13): error TS2717: Subsequent property declarations must have the same type.  Property 'progress' must be of type 'DetailedHTMLProps<ProgressHTMLAttributes<HTMLProgressElement>, HTMLProgressElement>', but here has type 'DetailedHTMLProps<ProgressHTMLAttributes<HTMLProgressElement>, HTMLProgressElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3186,13): error TS2717: Subsequent property declarations must have the same type.  Property 'q' must be of type 'DetailedHTMLProps<QuoteHTMLAttributes<HTMLQuoteElement>, HTMLQuoteElement>', but here has type 'DetailedHTMLProps<QuoteHTMLAttributes<HTMLQuoteElement>, HTMLQuoteElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3187,13): error TS2717: Subsequent property declarations must have the same type.  Property 'rp' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3188,13): error TS2717: Subsequent property declarations must have the same type.  Property 'rt' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3189,13): error TS2717: Subsequent property declarations must have the same type.  Property 'ruby' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3190,13): error TS2717: Subsequent property declarations must have the same type.  Property 's' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3191,13): error TS2717: Subsequent property declarations must have the same type.  Property 'samp' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3192,13): error TS2717: Subsequent property declarations must have the same type.  Property 'slot' must be of type 'DetailedHTMLProps<SlotHTMLAttributes<HTMLSlotElement>, HTMLSlotElement>', but here has type 'DetailedHTMLProps<SlotHTMLAttributes<HTMLSlotElement>, HTMLSlotElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3193,13): error TS2717: Subsequent property declarations must have the same type.  Property 'script' must be of type 'DetailedHTMLProps<ScriptHTMLAttributes<HTMLScriptElement>, HTMLScriptElement>', but here has type 'DetailedHTMLProps<ScriptHTMLAttributes<HTMLScriptElement>, HTMLScriptElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3194,13): error TS2717: Subsequent property declarations must have the same type.  Property 'section' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3195,13): error TS2717: Subsequent property declarations must have the same type.  Property 'select' must be of type 'DetailedHTMLProps<SelectHTMLAttributes<HTMLSelectElement>, HTMLSelectElement>', but here has type 'DetailedHTMLProps<SelectHTMLAttributes<HTMLSelectElement>, HTMLSelectElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3196,13): error TS2717: Subsequent property declarations must have the same type.  Property 'small' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3197,13): error TS2717: Subsequent property declarations must have the same type.  Property 'source' must be of type 'DetailedHTMLProps<SourceHTMLAttributes<HTMLSourceElement>, HTMLSourceElement>', but here has type 'DetailedHTMLProps<SourceHTMLAttributes<HTMLSourceElement>, HTMLSourceElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3198,13): error TS2717: Subsequent property declarations must have the same type.  Property 'span' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLSpanElement>, HTMLSpanElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLSpanElement>, HTMLSpanElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3199,13): error TS2717: Subsequent property declarations must have the same type.  Property 'strong' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3200,13): error TS2717: Subsequent property declarations must have the same type.  Property 'style' must be of type 'DetailedHTMLProps<StyleHTMLAttributes<HTMLStyleElement>, HTMLStyleElement>', but here has type 'DetailedHTMLProps<StyleHTMLAttributes<HTMLStyleElement>, HTMLStyleElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3201,13): error TS2717: Subsequent property declarations must have the same type.  Property 'sub' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3202,13): error TS2717: Subsequent property declarations must have the same type.  Property 'summary' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3203,13): error TS2717: Subsequent property declarations must have the same type.  Property 'sup' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3204,13): error TS2717: Subsequent property declarations must have the same type.  Property 'table' must be of type 'DetailedHTMLProps<TableHTMLAttributes<HTMLTableElement>, HTMLTableElement>', but here has type 'DetailedHTMLProps<TableHTMLAttributes<HTMLTableElement>, HTMLTableElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3205,13): error TS2717: Subsequent property declarations must have the same type.  Property 'template' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLTemplateElement>, HTMLTemplateElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLTemplateElement>, HTMLTemplateElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3206,13): error TS2717: Subsequent property declarations must have the same type.  Property 'tbody' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLTableSectionElement>, HTMLTableSectionElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLTableSectionElement>, HTMLTableSectionElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3207,13): error TS2717: Subsequent property declarations must have the same type.  Property 'td' must be of type 'DetailedHTMLProps<TdHTMLAttributes<HTMLTableDataCellElement>, HTMLTableDataCellElement>', but here has type 'DetailedHTMLProps<TdHTMLAttributes<HTMLTableDataCellElement>, HTMLTableDataCellElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3208,13): error TS2717: Subsequent property declarations must have the same type.  Property 'textarea' must be of type 'DetailedHTMLProps<TextareaHTMLAttributes<HTMLTextAreaElement>, HTMLTextAreaElement>', but here has type 'DetailedHTMLProps<TextareaHTMLAttributes<HTMLTextAreaElement>, HTMLTextAreaElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3209,13): error TS2717: Subsequent property declarations must have the same type.  Property 'tfoot' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLTableSectionElement>, HTMLTableSectionElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLTableSectionElement>, HTMLTableSectionElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3210,13): error TS2717: Subsequent property declarations must have the same type.  Property 'th' must be of type 'DetailedHTMLProps<ThHTMLAttributes<HTMLTableHeaderCellElement>, HTMLTableHeaderCellElement>', but here has type 'DetailedHTMLProps<ThHTMLAttributes<HTMLTableHeaderCellElement>, HTMLTableHeaderCellElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3211,13): error TS2717: Subsequent property declarations must have the same type.  Property 'thead' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLTableSectionElement>, HTMLTableSectionElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLTableSectionElement>, HTMLTableSectionElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3212,13): error TS2717: Subsequent property declarations must have the same type.  Property 'time' must be of type 'DetailedHTMLProps<TimeHTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<TimeHTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3213,13): error TS2717: Subsequent property declarations must have the same type.  Property 'title' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLTitleElement>, HTMLTitleElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLTitleElement>, HTMLTitleElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3214,13): error TS2717: Subsequent property declarations must have the same type.  Property 'tr' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLTableRowElement>, HTMLTableRowElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLTableRowElement>, HTMLTableRowElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3215,13): error TS2717: Subsequent property declarations must have the same type.  Property 'track' must be of type 'DetailedHTMLProps<TrackHTMLAttributes<HTMLTrackElement>, HTMLTrackElement>', but here has type 'DetailedHTMLProps<TrackHTMLAttributes<HTMLTrackElement>, HTMLTrackElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3216,13): error TS2717: Subsequent property declarations must have the same type.  Property 'u' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3217,13): error TS2717: Subsequent property declarations must have the same type.  Property 'ul' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLUListElement>, HTMLUListElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLUListElement>, HTMLUListElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3218,13): error TS2717: Subsequent property declarations must have the same type.  Property '"var"' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3219,13): error TS2717: Subsequent property declarations must have the same type.  Property 'video' must be of type 'DetailedHTMLProps<VideoHTMLAttributes<HTMLVideoElement>, HTMLVideoElement>', but here has type 'DetailedHTMLProps<VideoHTMLAttributes<HTMLVideoElement>, HTMLVideoElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3220,13): error TS2717: Subsequent property declarations must have the same type.  Property 'wbr' must be of type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>', but here has type 'DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3221,13): error TS2717: Subsequent property declarations must have the same type.  Property 'webview' must be of type 'DetailedHTMLProps<WebViewHTMLAttributes<HTMLWebViewElement>, HTMLWebViewElement>', but here has type 'DetailedHTMLProps<WebViewHTMLAttributes<HTMLWebViewElement>, HTMLWebViewElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3224,13): error TS2717: Subsequent property declarations must have the same type.  Property 'svg' must be of type 'SVGProps<SVGSVGElement>', but here has type 'SVGProps<SVGSVGElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3226,13): error TS2717: Subsequent property declarations must have the same type.  Property 'animate' must be of type 'SVGProps<SVGElement>', but here has type 'SVGProps<SVGElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3227,13): error TS2717: Subsequent property declarations must have the same type.  Property 'animateMotion' must be of type 'SVGProps<SVGElement>', but here has type 'SVGProps<SVGElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3228,13): error TS2717: Subsequent property declarations must have the same type.  Property 'animateTransform' must be of type 'SVGProps<SVGElement>', but here has type 'SVGProps<SVGElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3229,13): error TS2717: Subsequent property declarations must have the same type.  Property 'circle' must be of type 'SVGProps<SVGCircleElement>', but here has type 'SVGProps<SVGCircleElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3230,13): error TS2717: Subsequent property declarations must have the same type.  Property 'clipPath' must be of type 'SVGProps<SVGClipPathElement>', but here has type 'SVGProps<SVGClipPathElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3231,13): error TS2717: Subsequent property declarations must have the same type.  Property 'defs' must be of type 'SVGProps<SVGDefsElement>', but here has type 'SVGProps<SVGDefsElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3232,13): error TS2717: Subsequent property declarations must have the same type.  Property 'desc' must be of type 'SVGProps<SVGDescElement>', but here has type 'SVGProps<SVGDescElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3233,13): error TS2717: Subsequent property declarations must have the same type.  Property 'ellipse' must be of type 'SVGProps<SVGEllipseElement>', but here has type 'SVGProps<SVGEllipseElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3234,13): error TS2717: Subsequent property declarations must have the same type.  Property 'feBlend' must be of type 'SVGProps<SVGFEBlendElement>', but here has type 'SVGProps<SVGFEBlendElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3235,13): error TS2717: Subsequent property declarations must have the same type.  Property 'feColorMatrix' must be of type 'SVGProps<SVGFEColorMatrixElement>', but here has type 'SVGProps<SVGFEColorMatrixElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3236,13): error TS2717: Subsequent property declarations must have the same type.  Property 'feComponentTransfer' must be of type 'SVGProps<SVGFEComponentTransferElement>', but here has type 'SVGProps<SVGFEComponentTransferElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3237,13): error TS2717: Subsequent property declarations must have the same type.  Property 'feComposite' must be of type 'SVGProps<SVGFECompositeElement>', but here has type 'SVGProps<SVGFECompositeElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3238,13): error TS2717: Subsequent property declarations must have the same type.  Property 'feConvolveMatrix' must be of type 'SVGProps<SVGFEConvolveMatrixElement>', but here has type 'SVGProps<SVGFEConvolveMatrixElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3239,13): error TS2717: Subsequent property declarations must have the same type.  Property 'feDiffuseLighting' must be of type 'SVGProps<SVGFEDiffuseLightingElement>', but here has type 'SVGProps<SVGFEDiffuseLightingElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3240,13): error TS2717: Subsequent property declarations must have the same type.  Property 'feDisplacementMap' must be of type 'SVGProps<SVGFEDisplacementMapElement>', but here has type 'SVGProps<SVGFEDisplacementMapElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3241,13): error TS2717: Subsequent property declarations must have the same type.  Property 'feDistantLight' must be of type 'SVGProps<SVGFEDistantLightElement>', but here has type 'SVGProps<SVGFEDistantLightElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3242,13): error TS2717: Subsequent property declarations must have the same type.  Property 'feDropShadow' must be of type 'SVGProps<SVGFEDropShadowElement>', but here has type 'SVGProps<SVGFEDropShadowElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3243,13): error TS2717: Subsequent property declarations must have the same type.  Property 'feFlood' must be of type 'SVGProps<SVGFEFloodElement>', but here has type 'SVGProps<SVGFEFloodElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3244,13): error TS2717: Subsequent property declarations must have the same type.  Property 'feFuncA' must be of type 'SVGProps<SVGFEFuncAElement>', but here has type 'SVGProps<SVGFEFuncAElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3245,13): error TS2717: Subsequent property declarations must have the same type.  Property 'feFuncB' must be of type 'SVGProps<SVGFEFuncBElement>', but here has type 'SVGProps<SVGFEFuncBElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3246,13): error TS2717: Subsequent property declarations must have the same type.  Property 'feFuncG' must be of type 'SVGProps<SVGFEFuncGElement>', but here has type 'SVGProps<SVGFEFuncGElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3247,13): error TS2717: Subsequent property declarations must have the same type.  Property 'feFuncR' must be of type 'SVGProps<SVGFEFuncRElement>', but here has type 'SVGProps<SVGFEFuncRElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3248,13): error TS2717: Subsequent property declarations must have the same type.  Property 'feGaussianBlur' must be of type 'SVGProps<SVGFEGaussianBlurElement>', but here has type 'SVGProps<SVGFEGaussianBlurElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3249,13): error TS2717: Subsequent property declarations must have the same type.  Property 'feImage' must be of type 'SVGProps<SVGFEImageElement>', but here has type 'SVGProps<SVGFEImageElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3250,13): error TS2717: Subsequent property declarations must have the same type.  Property 'feMerge' must be of type 'SVGProps<SVGFEMergeElement>', but here has type 'SVGProps<SVGFEMergeElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3251,13): error TS2717: Subsequent property declarations must have the same type.  Property 'feMergeNode' must be of type 'SVGProps<SVGFEMergeNodeElement>', but here has type 'SVGProps<SVGFEMergeNodeElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3252,13): error TS2717: Subsequent property declarations must have the same type.  Property 'feMorphology' must be of type 'SVGProps<SVGFEMorphologyElement>', but here has type 'SVGProps<SVGFEMorphologyElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3253,13): error TS2717: Subsequent property declarations must have the same type.  Property 'feOffset' must be of type 'SVGProps<SVGFEOffsetElement>', but here has type 'SVGProps<SVGFEOffsetElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3254,13): error TS2717: Subsequent property declarations must have the same type.  Property 'fePointLight' must be of type 'SVGProps<SVGFEPointLightElement>', but here has type 'SVGProps<SVGFEPointLightElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3255,13): error TS2717: Subsequent property declarations must have the same type.  Property 'feSpecularLighting' must be of type 'SVGProps<SVGFESpecularLightingElement>', but here has type 'SVGProps<SVGFESpecularLightingElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3256,13): error TS2717: Subsequent property declarations must have the same type.  Property 'feSpotLight' must be of type 'SVGProps<SVGFESpotLightElement>', but here has type 'SVGProps<SVGFESpotLightElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3257,13): error TS2717: Subsequent property declarations must have the same type.  Property 'feTile' must be of type 'SVGProps<SVGFETileElement>', but here has type 'SVGProps<SVGFETileElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3258,13): error TS2717: Subsequent property declarations must have the same type.  Property 'feTurbulence' must be of type 'SVGProps<SVGFETurbulenceElement>', but here has type 'SVGProps<SVGFETurbulenceElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3259,13): error TS2717: Subsequent property declarations must have the same type.  Property 'filter' must be of type 'SVGProps<SVGFilterElement>', but here has type 'SVGProps<SVGFilterElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3260,13): error TS2717: Subsequent property declarations must have the same type.  Property 'foreignObject' must be of type 'SVGProps<SVGForeignObjectElement>', but here has type 'SVGProps<SVGForeignObjectElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3261,13): error TS2717: Subsequent property declarations must have the same type.  Property 'g' must be of type 'SVGProps<SVGGElement>', but here has type 'SVGProps<SVGGElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3262,13): error TS2717: Subsequent property declarations must have the same type.  Property 'image' must be of type 'SVGProps<SVGImageElement>', but here has type 'SVGProps<SVGImageElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3263,13): error TS2717: Subsequent property declarations must have the same type.  Property 'line' must be of type 'SVGProps<SVGLineElement>', but here has type 'SVGProps<SVGLineElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3264,13): error TS2717: Subsequent property declarations must have the same type.  Property 'linearGradient' must be of type 'SVGProps<SVGLinearGradientElement>', but here has type 'SVGProps<SVGLinearGradientElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3265,13): error TS2717: Subsequent property declarations must have the same type.  Property 'marker' must be of type 'SVGProps<SVGMarkerElement>', but here has type 'SVGProps<SVGMarkerElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3266,13): error TS2717: Subsequent property declarations must have the same type.  Property 'mask' must be of type 'SVGProps<SVGMaskElement>', but here has type 'SVGProps<SVGMaskElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3267,13): error TS2717: Subsequent property declarations must have the same type.  Property 'metadata' must be of type 'SVGProps<SVGMetadataElement>', but here has type 'SVGProps<SVGMetadataElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3268,13): error TS2717: Subsequent property declarations must have the same type.  Property 'mpath' must be of type 'SVGProps<SVGElement>', but here has type 'SVGProps<SVGElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3269,13): error TS2717: Subsequent property declarations must have the same type.  Property 'path' must be of type 'SVGProps<SVGPathElement>', but here has type 'SVGProps<SVGPathElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3270,13): error TS2717: Subsequent property declarations must have the same type.  Property 'pattern' must be of type 'SVGProps<SVGPatternElement>', but here has type 'SVGProps<SVGPatternElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3271,13): error TS2717: Subsequent property declarations must have the same type.  Property 'polygon' must be of type 'SVGProps<SVGPolygonElement>', but here has type 'SVGProps<SVGPolygonElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3272,13): error TS2717: Subsequent property declarations must have the same type.  Property 'polyline' must be of type 'SVGProps<SVGPolylineElement>', but here has type 'SVGProps<SVGPolylineElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3273,13): error TS2717: Subsequent property declarations must have the same type.  Property 'radialGradient' must be of type 'SVGProps<SVGRadialGradientElement>', but here has type 'SVGProps<SVGRadialGradientElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3274,13): error TS2717: Subsequent property declarations must have the same type.  Property 'rect' must be of type 'SVGProps<SVGRectElement>', but here has type 'SVGProps<SVGRectElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3275,13): error TS2717: Subsequent property declarations must have the same type.  Property 'stop' must be of type 'SVGProps<SVGStopElement>', but here has type 'SVGProps<SVGStopElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3276,13): error TS2717: Subsequent property declarations must have the same type.  Property 'switch' must be of type 'SVGProps<SVGSwitchElement>', but here has type 'SVGProps<SVGSwitchElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3277,13): error TS2717: Subsequent property declarations must have the same type.  Property 'symbol' must be of type 'SVGProps<SVGSymbolElement>', but here has type 'SVGProps<SVGSymbolElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3278,13): error TS2717: Subsequent property declarations must have the same type.  Property 'text' must be of type 'SVGProps<SVGTextElement>', but here has type 'SVGProps<SVGTextElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3279,13): error TS2717: Subsequent property declarations must have the same type.  Property 'textPath' must be of type 'SVGProps<SVGTextPathElement>', but here has type 'SVGProps<SVGTextPathElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3280,13): error TS2717: Subsequent property declarations must have the same type.  Property 'tspan' must be of type 'SVGProps<SVGTSpanElement>', but here has type 'SVGProps<SVGTSpanElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3281,13): error TS2717: Subsequent property declarations must have the same type.  Property 'use' must be of type 'SVGProps<SVGUseElement>', but here has type 'SVGProps<SVGUseElement>'.
node_modules/adminjs/node_modules/@types/react/index.d.ts(3282,13): error TS2717: Subsequent property declarations must have the same type.  Property 'view' must be of type 'SVGProps<SVGViewElement>', but here has type 'SVGProps<SVGViewElement>'.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -22,7 +22,8 @@
       "paths": {
         "react": ["node_modules/@types/react"],
         "styled-components": ["node_modules/@types/styled-components"]
-      }
+      },
+      "skipLibCheck": true
   },
   "include": [
       "./vendor-types/**/*",
@@ -31,6 +32,7 @@
   ],
   "exclude": [
       "./example-app/**/*",
+      "./node-modules",
       "./src/frontend/assets/**/*",
       "./src/frontend/components/**/*stories.tsx",
   ]

